cli/src/main/java/org/jboss/as/cli/gui/metacommand/ScriptAction.java

```diff
@@ -85,22 +85,11 @@ protected void runScript(File script) {
 
     // read the file as a list of text lines
     private List<String> getCommandLines(File file) {
-        List<String> lines = new ArrayList<String>();
-        BufferedReader reader = null;
         try {
-            reader = new BufferedReader(new FileReader(file));
-            String line = reader.readLine();
-            while (line != null) {
-                lines.add(line);
-                line = reader.readLine();
-            }
-        } catch (Throwable e) {
-            throw new IllegalStateException("Failed to process file " + file.getAbsolutePath(), e);
-        } finally {
-            StreamUtils.safeClose(reader);
+            return Files.readAllLines(file.toPath(), StandardCharsets.UTF_8);
+        } catch (IOException e1) {
+            return Collections.emptyList();
```

Comment:
The old code handled any exception by throwing ISE; the new code suppresses IOException.